src/utils/translation-helpers.ts

```diff
@@ -16,11 +16,11 @@ export const getPreferredLanguageCode = (language?: string) => {
 };
 
 // Get label based on selected language
-export const getLanguageString = (labels?: LanguageString) => {
+export const getLanguageString = (labels?: LanguageString, preferredLanguageCode?: string) => {
```

Comment:
sikkert lurt med en eksplisitt type her, ellers kan man få tekst på `nb` i stedet for `en` om man sender inn `eng` etc, siden `eng`, `nob`, o.l. er helt ukjent i dette tilfellet
```suggestion
export const getLanguageString = (labels?: LanguageString, preferredLanguageCode?: 'nb' | 'nn' | 'no' | 'en') => {
```